app/models/job.js

```diff
@@ -39,6 +39,7 @@ export default Model.extend(DurationCalculations, DurationAttributes, {
   queue: attr(),
   state: attr(),
   number: attr(),
+  permissions: alias('build.permissions'),
```

Comment:
No, this approach is incorrect. Job response has it's own `@permissions` hash, independent from the `build`'s one:

![image](https://user-images.githubusercontent.com/4066489/85158592-304cc500-b265-11ea-997d-f0f912b3384f.png)

It isn't used at the moment, that's why there isn't a property on the `job` model, but most likely, it will be at some point. 

Also `prioritize` permission is set on `build` level, not on `job`.

I'd recommend tweaking this property to cover both cases: https://github.com/travis-ci/travis-web/blob/epic-fast-track/app/components/repo-actions.js#L67

For example, you could extract `item.permissions.prioritize` to a separate property (e.g. `hasPrioritizePermission` and make it check for permission on both `build` and `job`:
```
hasPrioritizePermission: or('item.permissions.prioritize', 'item.build.permissions.prioritize')
```
Does it make sense to you?

Reply:
Sure @artursmirnov sure, implemented the same way now, kindly review the same thank you.